h/static/scripts/test/annotation-metadata-test.js

```diff
@@ -237,12 +237,12 @@ describe('annotation-metadata', function () {
 
   describe ('.isAnnotation', function () {
     it ('returns true if an annotation is a top level annotation', function () {
-      assert(annotationMetadata.isAnnotation({
+      assert.isArray(annotationMetadata.isAnnotation({
```

Comment:
This does not make sense, I would expect a function called `isAnnotation` to return a boolean, not an array.